control-plane/grpc/proto/v1/node/node.proto

```diff
@@ -61,6 +70,16 @@ enum NodeStatus {
   Offline = 2;
 }
 
+// Status of the Node
+enum DrainStatus {
+    // Node has no drain labels
+    NotDraining = 0;
+    // Node has at least 1 drain label and at least 1 nexus
```

Comment:
In future releases, we'll do full drain, at which point the semantics of this variable will change.
I wonder what we can do about this, maybe having different drain types?